src/muse/timeslices.py

```diff
@@ -320,27 +321,27 @@ def timeslice_projector(
                [0, 0, 1],
                [0, 0, 0]])
         Coordinates:
-          * finest_timeslice  (finest_timeslice) MultiIndex
-          - finest_semester   (finest_timeslice) object 'winter' 'winter' ... 'summer'
-          - finest_week       (finest_timeslice) object 'weekday' ... 'weekend'
-          - finest_day        (finest_timeslice) object 'day' 'night' ... 'night' 'dusk'
-          * timeslice         (timeslice) MultiIndex
-          - semester          (timeslice) object 'winter' 'winter' 'summer'
-          - week              (timeslice) object 'weekday' 'weekend' 'weekend'
-          - day               (timeslice) object 'allday' 'dusk' 'night'
+          * finest_timeslice  (finest_timeslice) object 80B MultiIndex
+          * finest_semester   (finest_timeslice) object 80B 'winter' ... 'summer'
+          * finest_week       (finest_timeslice) object 80B 'weekday' ... 'weekend'
+          * finest_day        (finest_timeslice) object 80B 'day' 'night' ... 'dusk'
+          * timeslice         (timeslice) object 24B MultiIndex
+          * semester          (timeslice) object 24B 'winter' 'winter' 'summer'
+          * week              (timeslice) object 24B 'weekday' 'weekend' 'weekend'
+          * day               (timeslice) object 24B 'allday' 'dusk' 'night'
 
         It is possible to give as input an array which does not have a timeslice of its
         own:
 
         >>> nots = DataArray([5.0, 1.0, 2.0], dims="a", coords={'a': [1, 2, 3]})
-        >>> timeslice_projector(nots, ref, transforms).T
-        <xarray.DataArray (timeslice: 1, finest_timeslice: 10)>
+        >>> timeslice_projector(nots, ref, transforms).T  # doctest: +SKIP
```

Comment:
Out of interest, why are various doctests being skipped? Do you get differences in the floating-point values on different platforms or something?

Reply:
Exactly. I do not want to just remove them since they can be illustrative, but as actual examples to run, they are less useful.